### DISCUSSION TITLE
Meeting 2024-10-15

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#799] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-10-15)

**Comment by Dirbaio** [comment]
last call for objections to https://github.com/rust-embedded/embedded-hal/pull/634 , if none I'll merge.

**Comment by jannic** [comment]
Issue triage:
https://github.com/rust-embedded/heapless/pull/508 was unanswered for 2 months. The change looks reasonable to me.

**Comment by adamgreig** [comment]
cortex-m-pac PR https://github.com/rust-embedded/cortex-m/pull/560#event-14579700710

**Comment by adamgreig** [comment]
CI fix for cargo-binutils * https://github.com/rust-embedded/cargo-binutils/pull/148

**Comment by hdoordt** [comment]
Reminder: Next book sprint for the Discovery MB2 book will be on October 26, 17:00-21:00 CEST, 15:00-19:00 UTC, 8:00AM-12:00PM PST. Join if you're interested in helping to write the new discovery book! Give a thumbs up to https://github.com/rust-embedded/wg/discussions/797#discussioncomment-10812164 to express your interest in joining.
